docs/src/main/asciidoc/kafka/kafka_overview.adoc

```diff
@@ -1049,7 +1049,7 @@ public class KafkaBinderHealthImplementation implements KafkaBinderHealth {
     }
 
 	// The document has an example on how to implement these checks
-	// https://www.baeldung.com/apache-kafka-check-server-is-running
+	// https://docs.spring.io/spring-kafka/reference/html/#kafka-validation
```

Comment:
This link is probably not what we want since that is unrelated. I suggest dropping the link altogether and let the users decide how to write the implementation.